File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Vehicle/CarResponseDto.ts
export class CarResponseDto {
    idCar: number;
    brand: string;
    model: string;
    patent: string;
    color: string;
    fuel: string;
    kml: string;
    gnc: boolean;
    deleted:boolean;
  
    constructor(
      idCar: number,
      brand: string,
      model: string,
      patent: string,
      color: string,
      fuel: string,
      kml: string,
      gnc: boolean,
      deleted:boolean
    ) {
      this.idCar = idCar;
      this.brand = brand;
      this.model = model;
      this.patent = patent;
      this.color = color;
      this.fuel = fuel;
      this.kml = kml;
      this.gnc = gnc;
      this.deleted = deleted;
    }
  }
  

File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/car/CarResponseDto.java
package ps.com.viajeros.dtos.car;

import jakarta.persistence.Column;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class CarResponseDto {
    private Long idCar;
    private String brand;
    private String model;
    private String patent;
    private String color;
    private String fuel;
    private String kml;
    private boolean gnc;
    private boolean deleted;
}
